### PR TITLE
updates chrome driver version (2.36)

### PR DESCRIPTION
**Pull request by hanagantig**
I tried to setup InstaPy on cloud server following the instructions on docs, and there was an error 

> ImportError: sys.meta_path is None, Python is likely shutting down

So looking through the traceback I've found this 

> InstaPyError: chromedriver 2.34 is not supported, expects 2.36+

Updating chromedriver solved the problem

### CODE REVIEW

**Comment by Tjorriemorrie** [comment]
My bad, I missed that. I wonder if there's a way to get the version automatically from https://chromedriver.storage.googleapis.com/LATEST_RELEASE rather than hard-coding it.

File: docs/How_To_DO_Ubuntu_on_Digital_Ocean.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ $ sudo rm google-chrome-stable_current_amd64.deb
 
 ```bash
 $ git clone https://github.com/timgrossmann/InstaPy.git
-$ wget "https://chromedriver.storage.googleapis.com/2.34/chromedriver_linux64.zip"
+$ wget "https://chromedriver.storage.googleapis.com/2.36/chromedriver_linux64.zip"
```
Comment (Tjorriemorrie) [inline docs/How_To_DO_Ubuntu_on_Digital_Ocean.md]:
```
latest_version=$(wget https://chromedriver.storage.googleapis.com/LATEST_RELEASE -O -)
wget https://chromedriver.storage.googleapis.com/${latest_version}/chromedriver_linux64.zip
```

**Review by Tjorriemorrie — CHANGES_REQUESTED** [review]
Googled around: you can try that. I cannot test it. Let me know if it works :)

File: docs/How_To_DO_Ubuntu_on_Digital_Ocean.md (hunk 1 above)
Comment (hanagantig) [reply docs/How_To_DO_Ubuntu_on_Digital_Ocean.md]: Yep! It works! ) At least on my cloud server )

**Comment by timgrossmann** [comment]
@hanagantig
Thank you for contributing!
Please update it with the provided "latest version"

**Comment by hanagantig** [comment]
@timgrossmann 
I've updated it.
I'm glad to participate!

**Pull request merged**